DOC-2223: TinyMCE 6.8.1 community release. (#3013)
tinymce/tinymce-docs: modules/ROOT/pages/changelog.adoc

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.8.1 - 2023-11-29

=== Improved
* Colorpicker now includes the Brightness/Saturation selector and hue slider in the keyboard navigable items.

=== Fixed

> * The functions `schema.isWrapper` and `schema.isInline` didn't exclude element names that started with `#` those should not be considered elements.

Reviewer: There where some alterations to the master branch on this one: https://github.com/tinymce/tinymce/blob/master/modules/tinymce/CHANGELOG.md

I could merge master into develop but we will do that tomorrow anyway when we do the release.

Author: ```suggestion
* The functions `schema.isWrapper` and `schema.isInline` did not exclude node names that started with `#` which should not be considered as elements. 
```
